src/figuresClasses.ts

```diff
@@ -1,19 +1,79 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
```

Comment:
please make those types more specific, we have limited number or colors and shapes